semeio/communication/semeio_script.py

```diff
@@ -72,11 +75,14 @@ def run_with_handler(self, *args, **kwargs):
         self.run = MethodType(run_with_handler, self)
 
     def _get_output_dir(self):
-        base_dir = self.ert().resConfig().model_config.getEnspath()
+        res_config = self.ert().resConfig()
+        base_dir = res_config.config_path
```

Comment:
This could probably be added to the `libres facade`, that requires a PR in `ert`, but think it is worth it, especially if we think this will be an option in the new API.

Reply:
I am slightly hesitant to add more to the `libres_facade` because we're using it in this PR. I do understand you that the argument is it useful enough to be part of the new API, and as such it's not "just because we need it here". But to be honest, I don't know the future of the `libres_facade`, nor how the API will evolve.

Unless there's a strong reason I say we should leave it. Possibly create an issue in ERT that would discuss around the `libres_facade` and argue that at least we've found a reason to include this option.

Reply:
I would be in the favor of at least creating an issue, because part of the point of `libres_facade` is to highlight what functions we are actually using?

Reply:
Agreed

Reply:
Or, now looking at the facade I see that it all has to do with internal memory related information. What we are using here is completely disk related, and disk relation is something we want to avoid. I would argue that it would be against the future implementation of libres to add any functionality related to disk.

What we instead want is to use the new storage for all of this once that is ready, and completely avoid anything to disk. And instead, if users need it, have a solution for them to export the data to disk and then have them explicitly state the location.